templates/paragraphs/paragraph--oe-content-row--variant-inpage_navigation.html.twig

```diff
@@ -9,7 +9,7 @@
 {{ attach_library('oe_theme/inpage_navigation') }}
 <div class="ecl-row">
```

Comment:
This template doesn't seem the same of https://ecl-twig.netlify.com/ec/?path=/story/components-navigation-inpage-navigation--default

Reply:
I have added wrapper (ecl-container).
Difference still exist but it is related to many aspects of integration. 
For instance we adding id for anchor for div wrapper (not for H1 tag as in example).
Also for many paragraphs we don't usage h1.
But in general inpage navigation currently work without big difference from demo (or I miss something).